src/main/java/com/aws/greengrass/componentmanager/builtins/S3Downloader.java

```diff
@@ -67,6 +68,11 @@ protected String getArtifactFilename() {
         return pathStrings[pathStrings.length - 1];
     }
 
+    @Override
+    public void cleanup(ComponentStore componentStore) throws Exception {
```

Comment:
This method should be implemented so it actually performs the cleanup. I think that the cleanup logic should be moved from the ComponentStore.